app/views/submissions/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% posts.map do |s| %>
```

Comment:
Use of `map` is interesting here. Did you mean to use `each`?

Reply:
I'd also rename `s` to `post` to be a bit more descriptive about what the current iteration is.